queries.sql

```diff
@@ -9,4 +9,45 @@ SELECT date_of_birth FROM animals WHERE name = 'Agumon' OR name = 'pikachu';
 SELECT name, escape_attempts FROM animals WHERE weight_kg > 10.5;
 SELECT * FROM animals WHERE neutered = true;
 SELECT * FROM animals WHERE NOT name = 'Gabumon';
-SELECT * FROM animals WHERE weight_kg BETWEEN  10.4 AND 17.3;
\ No newline at end of file
+SELECT * FROM animals WHERE weight_kg BETWEEN  10.4 AND 17.3;
+
+BEGIN;
+UPDATE animals
+SET species = 'unspecified';
+ROLLBACK;
+SELECT * FROM animals;
+
+BEGIN;
+UPDATE animals
+SET species = 'digimon'
+WHERE name LIKE '%mon';
+SELECT * FROM animals;
+
```

Comment:
- [ ] Please add `COMMIT` after the `SELECT * FROM animals;` query because the requirement is to commit this transaction so kindly add `COMMIT` :slightly_smiling_face: 
- [ ] Please add the `SELECT * FROM animals;` query after the `COMMIT` to verify that the change persists after the commit so kindly fix these issues :slightly_smiling_face: